python/Dockerfile

```
@@ -1,11 +1,16 @@
 FROM pangeo/base-image:2024.03.22
 
+ENV RUSTFLAGS="-C link-arg=-Wl,-rpath-link,$CONDA_PREFIX/lib -C link-arg=-Wl,-rpath,$CONDA_PREFIX/lib -L$CONDA_PREFIX/lib"
+
 USER root
 
 RUN wget https://code-server.dev/install.sh -O /tmp/install.sh && \
     sh /tmp/install.sh --version 4.19.1 && \
     rm /tmp/install.sh
 
+RUN export RUSTFLAGS="-C link-arg=-Wl,-rpath-link,$CONDA_PREFIX/lib -C link-arg=-Wl,-rpath,$CONDA_PREFIX/lib -L$CONDA_PREFIX/lib" && \
+    pip install git+https://github.com/opendatacube/odc-algo.git@add-rust-geomedian-impl
```

If I'm not mistaken, the $CONDA_PREFIX env variable is only available in the conda environment, so need to do `conda activate notebook` first? Also try using `python -m pip install` to be safe.

```suggestion
RUN conda activate notebook && \
    export RUSTFLAGS="-C link-arg=-Wl,-rpath-link,${CONDA_PREFIX}/lib -C link-arg=-Wl,-rpath,${CONDA_PREFIX}/lib -L${CONDA_PREFIX}/lib" && \
    python -m pip install git+https://github.com/opendatacube/odc-algo.git@add-rust-geomedian-impl
```

---

I'm pretty sure the conda environment is active here, and you can see that in the logs.

I'll change to this and see how it goes though, no worries.